src/components/drawer-content/drawer-content.tsx

```diff
@@ -252,6 +258,7 @@ const DrawerContent: React.FC<Props> = () => {
               width="100%"
             >
               { renderAccordinRow(`${strings.total}:`, totalHour, totalColor) }
+              { renderAccordinRow(`${strings.initialTime}:`, initialTimeHour, initialTimeColor) }
```

Comment:
Does initial values need to be on UI? Could you just calculate it to the total on the background?

Reply:
It is included in the total, it's just more intuitive to include all the factors that composes the total time since 
logged + initial_time - expected = total